mnist_igor_two.py

```diff
@@ -0,0 +1,253 @@
+import argparse
+import sys
+import time
+
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+from neuronpp.cells.cell import Cell
+from neuronpp.utils.network_status_graph import NetworkStatusGraph
+
+from agents.ebner_olfactory_agent import EbnerOlfactoryAgent
+from agents.ebner_olfactory_agent import WEIGHT
+from populations.ebner_hebbian_population import EbnerHebbianPopulation
+
+# print(sys.path)
+# sys.path.extend('/Users/igor/git/neuronpp/neuronpp')
+for e in sys.path: print(e)
+# print(sys.path)
+found = False
+for e in sys.path:
+    if 'NEURON-7.7/nrn/lib/python' in e:
+        found = True
+        break
+if not found:
+    print("NEURON-7.7 path not found in sys.path; exiting")
+    sys.exit(0)
+
+
+class EbOlAg(EbnerOlfactoryAgent):
```

Comment:
Naming should be consistent with other Agents. So name it Ebner...Agent

Reply:
Oh, I did not require the inclusion of mnist_igor_two.py into the master. It was only designed for you commenting on my comments whether I understood it correctly. And try some very minor changes into the architecture.
Nonetheless, the EbnerOlfactoryAgent() has some constants in it (numbers of hidden and inhibitory neurons) and it makes it harder to extend it easily.

Reply:
OK, that's fine for now. but if you want me to comment on something - maybe the best way is to use Github Gist in the future: https://gist.github.com